scss/variables/typography.js

```diff
@@ -10,7 +10,8 @@
 
 var color = "#222",
 	fontSize = "20px",
-	lineHeight = 1.5;
+	lineHeight = 1.5,
+	linkColor = "#4078c0";
 
```

Comment:
My plan is to apply something sort of like this: https://scotch.io/tutorials/aesthetic-sass-2-colors